pkg/cloud_provider/clientset/clientset.go

```diff
@@ -19,19 +19,28 @@ package clientset
 
 import (
 	"context"
+	"errors"
 	"fmt"
+	"time"
 
 	authenticationv1 "k8s.io/api/authentication/v1"
 	corev1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/informers"
 	"k8s.io/client-go/kubernetes"
+	listersv1 "k8s.io/client-go/listers/core/v1"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
 	"k8s.io/klog/v2"
 )
 
+const (
+	resyncDuration = time.Minute * 30
```

Comment:
can we make this a configurable flag (with default 30)?

Reply:
Done.